### ISSUE TITLE
scePsmfPlayerGetPsmfInfo parameters

### ISSUE THREAD

**Issue by unknownbrackets** [hrydgard/pspautotests#191]
Per hrydgard/ppsspp@e9d5eb6, we should validate which lib versions support this and have a test including the output.

-[Unknown]

**Comment by hrydgard** [comment]
Yup, this should be tested properly indeed. I just set that check to LocoRoco's requested version.
